src/org/gk/slicing/SlicingEngine.java

```diff
@@ -1284,6 +1534,43 @@ public static void main(String[] args) {
             String targetDbPort = properties.getProperty("slicingDbPort");
             if (targetDbPort == null || targetDbPort.trim().length() == 0)
                 targetDbPort = getInput("Please input the slice database port");
+            
+            // Compare database.
```

Comment:
Make sure to change compare to "previousSlice"

Reply:
Done.